pkgs/servers/mail/listmonk/vendor-hash.nix

```diff
@@ -0,0 +1 @@
+"sha256-qeBuDM3REUxgu3ty02d7qsULH04USE0JUvBrtVnW8vg="
```

Comment:
What are those files? For updating scripts we should use sed or something similar to update the hashes directly.

Reply:
I inspired myself from MailHog, which uses the same mechanism. But I can refactor this into their own files, it was just easier to share them with the `frontend.nix` file sometimes.

Reply:
Removed.